### ISSUE TITLE
HistoryViewerField reverts on a DataObject doesn't update field values 

### ISSUE THREAD

**Issue by thats4shaw** [silverstripe/silverstripe-versioned-admin#107]
When reverting to a revision via the `HistoryViewerField` on a DataObject, the field values on the `Main` tab don't update.  

If you refresh the page, the field values reflect the reverted version.

I followed https://docs.silverstripe.org/en/4/developer_guides/model/versioning/#using-the-history-viewer for my implementation.  This slightly differs to history viewer on the SiteTree which refreshes every time you hit the `Content` tab.

I had a brief chat to @ScopeyNZ on Slack regarding this and he suggested that:
You could hook into `onAfterRollback` to trigger a refresh in Javascript. You can whack something in state and then read that back on the event to click back to the content tab (using entwine)`

**Comment by robbieaverill** [comment]
That's true, it's a symptom of using React via the new FormSchema API and the old PHP form fields API in the same screen, tied together with entwine. It's not nice. IMO the fix would be to ensure that history tabs are always loaded _without_ PJAX, meaning navigating in and out of the tab would always trigger a window refresh. I know @chillu won't like this suggestion though 😆

**Comment by ScopeyNZ** [comment]
> You could hook into `onAfterRollback` to trigger a refresh in Javascript. You can whack something in state and then read that back on the event to click back to the content tab (using entwine)

Worth noting that a `onAfterRollback` hook is not currently present.

Perhaps we should have a go at reducing the boilerplate for `ModelAdmin`'s? Having something that refreshes the content tab could be included in this work.

**Comment by chillu** [comment]
> IMO the fix would be to ensure that history tabs are always loaded without PJAX, meaning navigating in and out of the tab would always trigger a window refresh. I know @chillu won't like this suggestion though

You guessed correctly ;) Its one of those issues which stem from the weird intermediate state between Entwine and React powered logic. I agree that it's an issue, but I fear that we might need to leave a fix until we can do this properly, with a fully React powered UI (and more smarts about data binding)

**Comment by kinglozzer** [comment]
Any update on this? Any way we can just inject a `window.reload()` somewhere?
